### PR TITLE
entities integration

### PR DESCRIPTION
**Pull request by willtp87**

### CODE REVIEW

File: includes/callbacks.inc [hunk 1]
```diff
@@ -39,6 +32,10 @@ function islandora_critical_edition_setup(AbstractObject $fedora_object) {
   $results['no_edit'] = !islandora_critical_edition_edit_cwrc_access($fedora_object);
   $results['page_count'] = count($pages);
 
+  $entitiy_collection = variable_get('islandora_entities_ingest_collection', 'islandora:entity_collection');
```
Comment (adam-vessey) [inline includes/callbacks.inc]: Not major, but the variable name is misspelled... "entitiy" vs "entity".

**Comment by adam-vessey** [comment]
Doesn't look like it's been touched in this pull request, but:

```
islandora_critical_edition.module:
 +62: [minor] Function comment short description must be on a single line
 +68: [minor] Invalid @return data type, expected bool but found boolean
 +83: [minor] Function comment short description must be on a single line
 +89: [minor] Invalid @return data type, expected bool but found boolean
 +112: [minor] Invalid @return data type, expected bool but found boolean
```

**Comment by adam-vessey** [comment]
Additional Coder issues:

```
sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/includes/callbacks.inc:
 +54: [minor] Use XHTML style  tags instead of 
 +63: [minor] Function comment short description must be on a single line
 +72: [minor] Functions must not contain multiple empty lines in a row; found 2 empty lines

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/includes/create.inc:
 +4: [minor] The third line in the file doc comment must contain a description and must not be indented
 +137: [minor] Function comment short description must be on a single line

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/includes/delete.inc:
 +4: [minor] The third line in the file doc comment must contain a description and must not be indented
 +8: [minor] Function comment short description must be on a single line
 +69: [minor] Function comment short description must be on a single line
 +94: [minor] Function comment short description must be on a single line

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/includes/manage.inc:
 +4: [minor] The third line in the file doc comment must contain a description and must not be indented

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/includes/utilities.inc:
 +4: [minor] The third line in the file doc comment must contain a description and must not be indented
 +37: [minor] Function comment short description must be on a single line

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/includes/view.inc:
 +4: [minor] The third line in the file doc comment must contain a description and must not be indented
 +35: [normal] The $string argument to t http://api.drupal.org/api/function/t/() should not begin or end with a space.

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_critical_edition/theme/theme.inc:
 +4: [minor] The third line in the file doc comment must contain a description and must not be indented
 +12: [minor] Hook implementations should not duplicate @param documentation
```

**Comment by willtp87** [comment]
I'm making a note of the coder issues in another ticket.

**Comment by adam-vessey** [comment]
Works... Coder issues concerning comments should be dealt with in Ontime ticket 1670.

**Pull request merged**